### PR TITLE
update application name to JUSTIFI

### PR DESCRIPTION
**Pull request by rmroot**
connects #154

### CODE REVIEW

File: src/app/core-components/welcome/welcome.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <div class="card col-lg-6 col-sm-10">
             <div class="card-body">
                 <div class="card-title display-5">
-                    Welcome to the Non-Energy Benefits Tool!
+                    Welcome to the JUSTIFI!
```
Comment (RLiNREL) [inline src/app/core-components/welcome/welcome.component.html]: I would say Welcome to JUSTIFI! or Welcome to the JUSTIFI tool!

File: src/app/core-components/welcome/welcome.component.html [hunk 2]
```diff
@@ -22,9 +22,7 @@
 
                             <fa-icon [icon]="faChevronRight"></fa-icon>
                             <p class="card-text ms-2">
-                                Use the setup wizard to create a new assessment. Here you can create a company and facility
-                                to
-                                add your new assessment to or pick from existing ones in your system.
+                                Use the setup wizard to enter data relating to companies, facilities and assessments.
```
Comment (RLiNREL) [inline src/app/core-components/welcome/welcome.component.html]: A little change and add a comma after facilities:
Use the setup wizard to enter data **about the** companies, facilities, and assessments.

File: package.json [hunk 3]
```diff
@@ -23,9 +23,9 @@
   "build": {
     "appId": "gov.nrel.neb-tool",
```
Comment (RLiNREL) [inline package.json]: Should we change to gov.nrel.justifi?

File: src/app/core-components/welcome/welcome.component.html
Comment (RLiNREL) [inline src/app/core-components/welcome/welcome.component.html:1]: The word "NEB-Tool" should be changed to "JUSTIFI" at the end of this html page
                                View help documents and tutorials to learn more about using the **NEB-Tool** application.

**Review by RLiNREL — CHANGES_REQUESTED** [review]
A little bit of a change to the wording, and a couple of places may need naming changes too. See the detailed comments.

File: package.json (hunk 3 above)
Comment (rmroot) [reply package.json]: I don't think this is actively used until we are doing releases and then it will most likely change but we can change it for now

**Review by RLiNREL — APPROVED** [review]

**Pull request merged**